README.md

```diff
@@ -179,6 +179,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
 | [Micro User Service](https://m3o.com/user) | User management and authentication | `apiKey` | Yes | No |
 | [MojoAuth](https://mojoauth.com) | Secure and modern passwordless authentication platform | `apiKey` | Yes | Yes |
+| [SAWO Labs](https://sawolabs.com) | Simplify login and improve user experience by integrating passwordless authentication in your app. Circumvent your bounce rates by 30%. | "apiKey" | Yes | Yes |
```

Comment:
Fix this, please:

- description should not end with .
- description should not exceed 100 characters (currently 135)
- auth value is not enclosed with \`backticks\`
- "apiKey" is not a valid Auth option

```suggestion
| [SAWO Labs](https://sawolabs.com) | Simplify login and improve user experience by integrating passwordless authentication in your app | `apiKey` | Yes | Yes |
```